.changeset/large-swans-join.md

```diff
@@ -0,0 +1,5 @@
+---
+'skuba': patch
+---
+
+template/koa-rest-api: Update `seek-koala` to prevent logger context mutation
```

Comment:
We should be able to do without this Changeset because it's a caret range; any `init`s from here will pick up the newest version.